.circleci/config.yml

```diff
@@ -46,6 +46,15 @@ references:
           - /^rc.*/
           - /^hotfix.*/
 
+  filter-rc-hotfix-branch: &filter-rc-hotfix-branch
+    filters:
+      tags:
+        ignore: /.*/
+      branches:
+        only:
+          - /^rc.*/
+          - /^hotfix.*/
+
```

Comment:
Make the filter reusable across all workflows. It is copied from `launch-all-app-builds-workflow`.

```
  launch-all-app-builds-workflow:
    # using do-builds param to select which of the two workflows using the build-on-tag-filters
    # should run; should be this one that just launches the builds or the one actually doing the builds
    unless: << pipeline.parameters.do-builds >>
    jobs:
      - conditionally-launch-build-and-store-all-job: &build-and-store-filters
          filters:
            branches:
              only:
                - /^rc.*/
                - /^hotfix.*/
 ```